x-pack/plugins/cases/public/components/add_comment/index.tsx

```diff
@@ -174,6 +175,9 @@ export const AddComment = React.memo(
         // eslint-disable-next-line react-hooks/exhaustive-deps
       }, [focusOnContext]);
 
+      const isDisabled =
+        isLoading || !comment?.trim().length || comment.trim().length > MAX_COMMENT_LENGTH;
```

Comment:
I think we should not disable the button if the user does not touch the form, right? You have to use `form.isValid` to check if the user has interacted with the form. If `undefined` then the form validation has not been checked yet. We can do it when we address https://github.com/elastic/kibana/issues/161461.